s/web/templates/index.scss

```diff
@@ -18,6 +18,10 @@ html {
 	font-size: 16px;
 }
 
+body {
+	display: grid;
+}
```

Comment:
`display: grid` on the whole body seems like a mistake?

Reply:
It wasn't, I needed it when I had the `lookpad` and `stickpad` side-by-side` (for some reason, the responsiveness on mobile was not okay, so that fixed it), but after rebasing and letting go of that code, I should've removed it.